unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Minimap/MinimapHUDController.cs

```diff
@@ -93,4 +93,10 @@ public void SetVisibility(bool visible)
     {
         view.SetVisibility(visible);
     }
+
+    public void AddUsersAroundIndicator(UsersAroundListHUDController controller)
```

Comment:
In order to follow the good practice of improve our documentation, from some time now we try to add a basic documentation to all the new classes and public methods (not including tests). Could you add a basic description as header about what this function do? It will help to anyone who need to call it, seeing the description throught the IDE intellisense.

It would be fine if you could apply this to all new classes / public methods you have created in this PR (at least in all those places where you think it would be useful or has sense).